Makefile

```diff
@@ -118,13 +121,17 @@ thrift-glean-hs:
 	# index goes in a subdir, so do it separately
 	$(THRIFT_COMPILE) --hs glean/if/index.thrift \
 		-o glean/if/index
+	# glass goes in a subdir, so do it separately
+	$(THRIFT_COMPILE) --hs glean/glass/if/glass.thrift \
+		-o glean/glass/if/glass
+
 
 .PHONY: thrift-schema-hs
 thrift-schema-hs:
 	for s in $(SCHEMAS); do \
 		$(THRIFT_COMPILE) --hs \
 			glean/schema/thrift/$$s.thrift \
-			-o glean/schema/thrift; \
+			-o glean/schema/thrift & \
```

Comment:
Wouldn't this cause the rule to be considered complete before all code generation has finished?